doc/Portfolio.txt

```diff
@@ -0,0 +1,71 @@
+# Portfolio
```

Comment:
I don't know if this text file is necessary, because this should be part of the code doucmentation. There you can find the files and the comments, or? I would only add the domain stories.